sdk/signalr/arm-signalr/review/arm-signalr.api.md

```diff
@@ -453,15 +462,16 @@ export interface SignalR {
     beginDeleteAndWait(resourceGroupName: string, resourceName: string, options?: SignalRDeleteOptionalParams): Promise<void>;
     beginRegenerateKey(resourceGroupName: string, resourceName: string, parameters: RegenerateKeyParameters, options?: SignalRRegenerateKeyOptionalParams): Promise<SimplePollerLike<OperationState<SignalRRegenerateKeyResponse>, SignalRRegenerateKeyResponse>>;
     beginRegenerateKeyAndWait(resourceGroupName: string, resourceName: string, parameters: RegenerateKeyParameters, options?: SignalRRegenerateKeyOptionalParams): Promise<SignalRRegenerateKeyResponse>;
-    beginRestart(resourceGroupName: string, resourceName: string, options?: SignalRRestartOptionalParams): Promise<SimplePollerLike<OperationState<void>, void>>;
-    beginRestartAndWait(resourceGroupName: string, resourceName: string, options?: SignalRRestartOptionalParams): Promise<void>;
+    beginRestart(resourceGroupName: string, resourceName: string, options?: SignalRRestartOptionalParams): Promise<SimplePollerLike<OperationState<SignalRRestartResponse>, SignalRRestartResponse>>;
```

Comment:
Is this change reflected in the changelog ?

Reply:
here:https://github.com/Azure/azure-sdk-for-js/pull/26995/files#diff-b3b6ac9434f9e000a346adeb4554adb667c6bef669798ddece3401609eb80291R31